website/static/_redirects

```diff
@@ -12,3 +12,17 @@
 
 # Changed blog post dates
 /blog/2021/03/11/version-0.64   /blog/2021/03/12/version-0.64
+
+
+
+
+/docs/architecture-overview             /architecture/overview
+/docs/fabric-renderer                   /architecture/fabric-renderer
+/docs/render-pipeline                   /architecture/render-pipeline
+/docs/xplat-implementation              /architecture/xplat-implementation
+/docs/view-flattening                   /architecture/view-flattening
+/docs/threading-model                   /architecture/threading-model
+/docs/architecture-glossary             /architecture/glossary
+
+
+
```

Comment:
```suggestion
# Architecture pages move to unversioned docs
/docs/architecture-overview             /architecture/overview
/docs/fabric-renderer                   /architecture/fabric-renderer
/docs/render-pipeline                   /architecture/render-pipeline
/docs/xplat-implementation              /architecture/xplat-implementation
/docs/view-flattening                   /architecture/view-flattening
/docs/threading-model                   /architecture/threading-model
/docs/architecture-glossary             /architecture/glossary

/docs/next/architecture-overview             /architecture/overview
/docs/next/fabric-renderer                   /architecture/fabric-renderer
/docs/next/render-pipeline                   /architecture/render-pipeline
/docs/next/xplat-implementation              /architecture/xplat-implementation
/docs/next/view-flattening                   /architecture/view-flattening
/docs/next/threading-model                   /architecture/threading-model
/docs/next/architecture-glossary             /architecture/glossary

/docs/0.66/architecture-overview             /architecture/overview
/docs/0.66/fabric-renderer                   /architecture/fabric-renderer
/docs/0.66/render-pipeline                   /architecture/render-pipeline
/docs/0.66/xplat-implementation              /architecture/xplat-implementation
/docs/0.66/view-flattening                   /architecture/view-flattening
/docs/0.66/threading-model                   /architecture/threading-model
/docs/0.66/architecture-glossary             /architecture/glossary
```

A chunk of new redirects there, but they should cover all of the cases IIRC.

Also what's the difference between those redirect and https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects ?

Reply:
<img width="987" alt="image" src="https://user-images.githubusercontent.com/749374/157672229-e15e9258-57ab-4285-af63-88c3ea03d4ca.png">

Those are server-side Netlify redirects (not related to Docusaurus at all, it is CDN config): https://docs.netlify.com/routing/redirects/